packages/sveltekit/package.json

```diff
@@ -35,7 +35,6 @@
     "@sveltejs/kit": "^1.11.0",
     "rollup": "^3.20.2",
     "svelte": "^3.44.0",
-    "typescript": "^4.9.3",
```

Comment:
m: I'm not sure if we should remove this. Right now ^4.9.3 would basically resolve to our global 4.9.5 version, right? But if we go to 5.x in the future, we'd also implicitly raise the version here which _potentially_ could cause problem with by then older SvelteKit versions 🤔

Reply:
as this is just a dev dependency, it _shouldn't_ affect the output, just how the types are built? Unless this somehow works differently for sveltekit?

Reply:
Good point. Let's go with this, we can revisit anyway.